views/pages/SignInPage.jsx

```diff
@@ -1,23 +1,23 @@
-import React, { useState, useContext } from 'react'
```

Comment:
should these files be nested in a `SignInPage` directory like the other component directories?

Reply:
[9d7c4dc](https://github.com/AMP-SCZ/dpdash/pull/468/commits/9d7c4dc05024fa07956aaa578a84f844f590db07)